misc.c

```diff
@@ -1038,7 +1038,7 @@ OVPNMsgWait(DWORD timeout, HWND hdlg)
         if (MsgWaitForMultipleObjectsEx(0, NULL, end - now, QS_ALLINPUT, MWMO_INPUTAVAILABLE) == WAIT_OBJECT_0)
         {
             MSG msg;
-            while(PeekMessage(&msg, NULL, 0, 0, PM_REMOVE))
+            while(PeekMessage(&msg, NULL, 0, WM_MANAGEMENT - 1, PM_REMOVE))
```

Comment:
I haven't tested, but it looks like this may affect exit processing. We wait for openvpn.exe to disconnect during exit, waiting here for up to 5 seconds. If management events are not processed during that wait, we'll not get OnStop() called.

Simplest option may be to revert to Sleep() in

```
else if (strncmp(pos, "INFO:", 5) == 0)
{
    /* delay until management interface accepts input */
    OVPNMsgWait(100, c->hwndStatus);
...
```
A 100 msec sleep may not appear  a stall in the UI. Also its limited to a single status thread, not the main thread.